### DISCUSSION TITLE
Feedback/Honor system

### DISCUSSION THREAD

**Discussion by eliasfaltin** [GECO-Project-App/facilitation-app#81] [1 - Ideas, Suggestions, & Requests]
@Cristian-Nass brought up this idea:
As a standalone exercise that can be used at the end of a meeting (post or pre-checkout @Emmy-Emberverse ?), we could offer a "kudos" "honor" "feedback" exercise.

### Proposal
The exercise should:
- allow people to give honest and constructive feedback to team members
- ensure no one is "left out"
- be done by everyone on their own phone separately via a shared room code (which means it is dependant on the "team"/"multiplayer" version of the app -> discussion around that is to come).

### Inspiration
In League of Legends, after the game is won or lost, you (as the player) are presented with a screen showing your four team mates. You can award one of your teammates one of [three types of honors](https://leagueoflegends.fandom.com/wiki/Honor#Honoring_players):
- stayed cool (= was pleasant to play with)
- great shotcalling (= great strategy + leadership)
- GG <3 (= single-handedly carried me to a win)
<details><summary>Here is a screenshot of this screen</summary>
<p>
![chrome_2018_01_11_12_45_10](https://github.com/user-attachments/assets/19f4e6c5-8649-45e7-8164-73fff0d36442)
</p>
</details> 

Ray Dalio, in his book [Principles](https://en.wikipedia.org/wiki/Principles_(book)) outlines the "Dot Collector" application they use internally at Bridgewater (an organization famous for their radical transparency).
- [Here](https://youtu.be/tUu-6UxuGr8) is a video from Dalio explaining the tool.
- [Here](https://principlesus.com/dot-collector-real-time-feedback/) is the sales page from his "Principles Us" product where you can buy the dot collector and his personality test tool.

**Comment by Emmy-Emberverse** [comment]
That's a fun idea!

What's the connection to the dot voting?

I like the idea of a kudos/honor exercise at the end. For me that fits
under 'checkout'. I personally prefer spoken open feedback to the group as
a whole more than categorized feedback to individuals, but I guess that
also means we need to mention the I-statement somewhere so that people
don't start attacking each other.

On Tue, Sep 24, 2024 at 4:08 PM eliasfaltin ***@***.***>
wrote:

> @Cristian-Nass <https://github.com/Cristian-Nass> brought up this idea:
> As a standalone exercise that can be used at the end of a meeting (post or
> pre-checkout @Emmy-Emberverse <https://github.com/Emmy-Emberverse> ?), we
> could offer a "kudos" "honor" "feedback" exercise.
> Proposal
>
> The exercise should:
>
>    - allow people to give honest and constructive feedback to team members
>    - ensure no one is "left out"
>    - be done by everyone on their own phone separately via a shared room
>    code (which means it is dependant on the "team"/"multiplayer" version of
>    the app -> discussion around that is to come).
>
> Inspiration
>
> In League of Legends, after the game is won or lost, you (as the player)
> are presented with a screen showing your four team mates. You can award one
> of your teammates one of three types of honors
> <https://leagueoflegends.fandom.com/wiki/Honor#Honoring_players>:
>
>    - stayed cool (= was pleasant to play with)
>    - great shotcalling (= great strategy + leadership)
>    - GG <3 (= single-handedly carried me to a win)
>
> Here is a screenshot of this screen
>
> ![chrome_2018_01_11_12_45_10](
> https://github.com/user-attachments/assets/19f4e6c5-8649-45e7-8164-73fff0d36442)
>
>
> Ray Dalio, in his book Principles
> <https://en.wikipedia.org/wiki/Principles_(book)> outlines the "Dot
> Collector" application they use internally at Bridgewater (an organization
> famous for their radical transparency).
>
>    - Here <https://youtu.be/tUu-6UxuGr8> is a video from Dalio explaining
>    the tool.
>    - Here <https://principlesus.com/dot-collector-real-time-feedback/> is
>    the sales page from his "Principles Us" product where you can buy the dot
>    collector and his personality test tool.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/GECO-Project-App/facilitation-app/discussions/81>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BLNOJCN2KGNOAMRH4KT555TZYFW6HAVCNFSM6AAAAABOYM3FP2VHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZXGIZTANRRGU>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
*EMMY PATER* | Regenerative Learning Developer & Facilitator
Mobile: +31629295478
Email: ***@***.***
Get in touch on LinkedIn <https://www.linkedin.com/in/emmy-pater-79794776/>

**Comment by aripuguh** [comment]
Fun idea @eliasfaltin ! Can definitely create visual assets/page for it once we are good to go.
